todo_cli/complete.js

```diff
@@ -0,0 +1,28 @@
+const complete = (fs, num, fn) => {
+  const x = +num;
```

Comment:
Is there a reason for the `+` operator here?

Use descriptive variable names where possible.

If the variable is not mutated/used in an expression elsewhere/renamed something more readable, consider using the param directly.

```
const myFunc = (descriptiveParam) => {
  if (descriptiveParam != 0) { doSomething() }
}
```

rather than

```
const myFunc = (descriptiveParam) => {
  const x = descriptiveParam
  if (x != 0) { doSomething() }
}
```